hipchat.go

```diff
@@ -201,9 +204,14 @@ func (c *Client) listen() {
 				continue
 			}
 
+			// parse time
+			time_pieces := strings.Split(attr["ts"], ".")
```

Comment:
oh for a bot, if you connect to hipchat with the resource of "bot" you wont get sent the chat history. if you connect as any other resource you will. try that first and let me know. this is an example of it: https://github.com/daneharrigan/hipchat/blob/master/example/hello.go#L11